_data/guidesnavswanlake.yml

```
@@ -39,6 +39,21 @@
     - title: Writing Tests
       url: /learn/testing-ballerina-code/writing-tests/
       active: writing-tests
+    - title: Structuring the tests
+      url: /learn/testing-ballerina-code/structuring-the-tests/
+      active: structuring-the-tests
+    - title: Configuring Tests
```

```suggestion
    - title: Configuring Tests
```
If we say `Structuring the Tests`, this should also be `Configuring the Tests`. Please make the titles consistent.

---

Changed `Structuring the Tests` to `Structuring Tests`